config.defaults.yaml

```diff
@@ -1344,9 +1344,12 @@ feature_generation:
     CPU:
       - LS
     GPU:
-      - ECE
-      - EAOV
-      - ELS
+      # If ELS_ECE_EAOV is set, period finding will be run using a nested approach which passes the top N ELS and ECE periods to EAOV.
+      # Individual ELS, ECE and EAOV periods will also be saved.
+      - ELS_ECE_EAOV
```

Comment:
Maybe this is better as an option rather than a default until some study is done? Or are you confident this has improved performance?

Reply:
That might be a good idea, especially since my preliminary check showed minimal differences between the golden set periods and the new algorithm's performance as compared to the full EAOV run. 92% of the ELS_ECE_EAOV periods are within 10% of the corresponding period from the full EAOV run.

Reply:
Ok. If you look at them and decide those 10% are mostly better with this new method, then you might go for it.

Reply:
Sounds good!